docs/contributing/docs-and-website-components.md

```diff
@@ -0,0 +1,409 @@
+---
+title: Docs and Website Components
+tableOfContentsDepth: 2
+---
+
+import LayerModel from "../../www/src/components/layer-model"
+import HorizontalNavList from "../../www/src/components/horizontal-nav-list"
+import Breadcrumb from "../../www/src/components/docs-breadcrumb"
+import { itemListContributing } from "../../www/src/utils/sidebar/item-list"
+import TableOfContents from "../../www/src/components/docs-table-of-contents"
+
+<div onClick={console.log({ itemListContributing, props })} />
```

Comment:
Should this DIV be in here, since it isn't accessible? (I'm wondering if it was leftover from debugging?)